packages/utils/src/random-string.ts

```diff
@@ -0,0 +1,5 @@
+import { nanoid } from 'nanoid';
+
+const randomString = (): string => nanoid().replace(/[_-]/g, '').toLowerCase();
+
+export default randomString;
```

Comment:
Remove, as suggested in the other comment.

Reply:
Will do